packages/sdk/src/SshConfigUtils.ts

```diff
@@ -71,7 +71,9 @@ export class SshConfigUtils {
                                     session.user = value;
                                     break;
                                 case "identityfile":
-                                    session.privateKey = value;
+                                    session.privateKey = value.startsWith("~")
+                                        ? path.normalize(path.resolve(path.join(os.homedir(), value.slice(2))))
+                                        : path.normalize(path.resolve(value));
```

Comment:
Not sure if the `path.resolve` is necessary here, also this could be simplified:
```suggestion
                                    session.privateKey = path.normalize(value.startsWith("~")
                                        ? path.join(os.homedir(), value.slice(2)) : value);
```